src/commands/createNewProject/ProjectCreateStep/PowerShellProjectCreateStep.ts

```diff
@@ -41,5 +51,10 @@ export class PowerShellProjectCreateStep extends ScriptProjectCreateStep {
         if (await confirmOverwriteFile(profileps1Path)) {
             await fse.writeFile(profileps1Path, profileps1);
         }
+
+        const requirementspsd1Path: string = path.join(wizardContext.projectPath, requirementspsd1FileName);
+        if (await confirmOverwriteFile(requirementspsd1Path)) {
```

Comment:
We have some existing tests for creating a project. Just search the 'test' folder in this repo for `profile.ps1` and copy each line and replace it with `requirements.psd1`

Reply:
I have added a new check to ensure ```requirements.psd1``` gets created.